docs/app/references/migration-guide.mdx

```diff
@@ -20,6 +20,69 @@ Cypress requires [Node.js](https://nodejs.org/en) in order to install the Cypres
 Node.js versions 18 and 23 are no longer supported.
 [See Node's release schedule](https://github.com/nodejs/Release).
 
+### Webpack `4` is no longer supported
+
+Cypress is no longer supporting Webpack `4` as it is no longer maintained by the core Webpack team and Webpack `5` has been available since Q4 2020. This includes Webpack `4` support for:
+
+- `@cypress/webpack-batteries-included-preprocessor` and `@cypress/webpack-preprocessor` for end-to-end testing. This is mostly a passive change unless you are using a specific webpack configuration to preprocess your tests, which is uncommon.
+- `@cypress/webpack-dev-server` for component testing. This use case is more common and will require an to update to Webpack `5`.
+
+#### To continue using Webpack `4`
+
+##### End-to-End Testing
+
+If you haven't been able to migrate away from Webpack `4` and still need to be able to run your end-to-end tests with Webpack `4`, you can install the following packages independently:
+
+```sh
+npm install --save-dev @cypress/webpack-batteries-included-preprocessor@3.x.x @cypress/webpack-preprocessor@6.x.x
+```
+
+and configure the preprocessor within your `cypress.config.js` or `cypress.config.ts` file:
+
+```js
+import { defineConfig } from 'cypress'
+import webpackPreprocessor from '@cypress/webpack-batteries-included-preprocessor'
+
+export default defineConfig({
+  e2e: {
+    setupNodeEvents(on, config) {
+      on('file:preprocessor', webpackPreprocessor())
+    },
+  },
+})
+```
+
+More information on how to configure the preprocessor can be found in the [Preprocessors API documentation](https://docs.cypress.io/api/node-events/preprocessors-api#Usage), the [Webpack Batteries Included Preprocessor documentation](https://github.com/cypress-io/cypress/blob/v14.3.2/npm/webpack-batteries-included-preprocessor/README.md), and [Webpack Preprocessor documentation](https://github.com/cypress-io/cypress/blob/v14.3.2/npm/webpack-preprocessor/README.md).
```

Comment:
```suggestion
More information on how to configure the preprocessor can be found in the [Preprocessors API documentation](/api/node-events/preprocessors-api#Usage), the [Webpack Batteries Included Preprocessor documentation](https://github.com/cypress-io/cypress/blob/v14.3.2/npm/webpack-batteries-included-preprocessor/README.md), and [Webpack Preprocessor documentation](https://github.com/cypress-io/cypress/blob/v14.3.2/npm/webpack-preprocessor/README.md).
```

Change to relative. 
Not sure if the other references should be bound to a Cypress v14 version.

Reply:
I forgot that we actually tag npm published packages on the published git commit. For example: https://github.com/cypress-io/cypress/releases/tag/%40cypress%2Fwebpack-preprocessor-v6.0.4. I will get these references updated